style.css

```diff
@@ -34,19 +42,23 @@ header {
 }
 
 .mainpage {
-  display: flex;
   flex-flow: column wrap;
-  align-items: center;
-  text-align: center;
+  background-image: url(./bg/9.png), url(/bg/3.png), url(bg/1.png), url(bg/8.png), url(bg/4.png), url(/bg/7.png);
+  background-repeat: no-repeat;
```

Comment:
- [ ] **REQUIRED:** Your background image is not covering the full width, there is little space on both right and left side, please kindly set the width 100wh.